CHANGELOG.md

```diff
@@ -1,8 +1,14 @@
+# v3.1.1
+* [BUGFIX] Wait for informer cache to be fully populates before interrogating it (#218)
```

Comment:
I agree but the versioning used for 3.1.0 doesn't suggest it was a bug, but rather a feature, otherwise it should have been 3.0.1. So I simply added the missing changelog respecting the choice of version